messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -89,7 +91,8 @@ class Coordinator {
     private final long claimExtensionThreshold;
     private final Clock clock;
     private final int maxClaimedSegments;
-
+    private final int initialSegmentCount;
+    private final Function<StreamableMessageSource<TrackedEventMessage<?>>, TrackingToken> initialToken;
```

Comment:
Super nit: I deliberately had an empty line between the fields configured through the `Builder` and the fields this class constructs itself. By no means a necessity to reintroduce, just wanted to clarify the intent of the empty line. :sweat_smile:

Reply:
added again.